src/main/java/cs2103/v15_1j/jimjim/command/AddCommand.java

```diff
@@ -46,7 +46,8 @@ public UIFeedback undo(ControllerStates conStates) {
     @Override
     public UIFeedback execute(ControllerStates conStates) {
         conStates.masterList.add(taskEvent);
-        
+        conStates.resetRedoHistory();
```

Comment:
This way of doing it is a bit laborious / not nice to me, as you have to add this to every single command. How about checking in `JJController` whether the command is UndoCommand, if not clear the redo stack?